core/nuxeo-operation.js

```diff
@@ -330,27 +393,90 @@ import './nuxeo-connection.js';
         });
     }
 
+    /**
+     * Handler to abort bulk operations executed through the BAF
+     */
+    _abort(commandId) {
+      // CHECK - is this enough to abort the previous request?
```

Comment:
Reminder to avoid forgetting this check.

Reply:
I confirmed that it is not necessary, removing it 👍 thanks